src/test/java/blackjack/domain/card/CardsTest.java

```diff
@@ -0,0 +1,62 @@
+package blackjack.domain.card;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CardsTest {
+    private Cards cards;
+    private Card king;
+    private Card queen;
+    private Card ace;
+    private Card two;
+
+    @BeforeEach
+    @Test
+    void setUp() {
+        cards = new Cards();
+        king = new Card(Suit.CLUB, Denomination.KING);
+        queen = new Card(Suit.CLUB, Denomination.QUEEN);
+        ace = new Card(Suit.CLUB, Denomination.ACE);
+        two = new Card(Suit.CLUB, Denomination.TWO);
+    }
+
+    @DisplayName("가진 패의 점수를 계산할 수 있다.")
+    @Test
+    void getScoreTest() {
+        cards.hit(king);
+        cards.hit(two);
+        assertThat(cards.getScore()).isEqualTo(12);
+    }
+
+    @DisplayName("Ace가 섞여 있을 때 최선의 점수를 계산할 수 있다.")
+    @Test
+    void getMaximunScoreTest() {
+        cards.hit(ace);
+        cards.hit(two);
+        assertThat(cards.getScore()).isEqualTo(13);
+
+        cards.hit(king);
+        assertThat(cards.getScore()).isEqualTo(13);
+    }
+
+    @DisplayName("블랙잭 여부를 판별할 수 있다.")
+    @Test
+    void isBlackJackTest() {
+        cards.hit(king);
+        cards.hit(ace);
+        assertTrue(cards.isBlackJack());
+    }
+
+    @DisplayName("버스트 여부를 판별할 수 있다.")
+    @Test
+    void isBustTest() {
+        cards.hit(king);
+        cards.hit(queen);
+        cards.hit(two);
+        assertTrue(cards.isBust());
```

Comment:
```suggestion
        cards.hit(king);
        cards.hit(queen);
        cards.hit(two);

        assertTrue(cards.isBust());
```

이렇게 줄바꿈이 들어가면 테스트 코드가 무엇을 검증하는지 한눈에 들어와서 더 좋아요 😉

Reply:
[커밋](https://github.com/tributetothemoon/java-blackjack/commit/30db48acdf7498d8e623d3e1d3e7ffc827c2d9dd): 개행문자를 추가해주는 편이 맞는 것이군요. 감사합니다😉